config/runtime.exs

```diff
@@ -252,7 +235,12 @@ if config_env() == :dev do
     static_bucket: System.fetch_env!("AWS_S3_BUCKET_STATIC"),
     static_cdn: System.fetch_env!("STATIC_CDN")
 
-  config :t, T.Media.Static, disabled?: false
+  config :t, T.Media.Static, disabled?: !!System.get_env("DISABLE_MEDIA")
+  config :t, T.Feeds.SeenPruner, disabled?: !!System.get_env("DISABLE_SEEN_PRUNER")
+  config :t, T.Matches.MatchExpirer, disabled?: !!System.get_env("DISABLE_MATCH_EXPIRER")
+
+  config :t, T.PushNotifications.ScheduledPushes,
+    disabled?: !!System.get_env("DISABLE_SCHEDULED_PUSHES")
```

Comment:
These are not disabled by default. And they are disabled if a relevant env var is set to anything `export DISABLE_MEDIA=1`.

Reply:
I disabled all these background workers to avoid accidentally sending someone a push notification or something like that (although it shouldn't happen since I'm using sandboxed creds to apns).